### PR TITLE
added changelog.md via auto-changelog

### PR DESCRIPTION
**Pull request by Spotts9**
https://github.com/react-native-elements/react-native-elements/issues/2626

### CODE REVIEW

**Review by pranshuchittora — COMMENTED** [review]
Can you an npm script for the following so that we can generate this before release or via CI. And document this as well

**Comment by Spotts9** [comment]
Followed [doc for Auto-changelog](https://www.npmjs.com/package/auto-changelog) to setup npm script.  Happy to document but not sure where / how.

File: package.json [hunk 1]
```diff
@@ -69,7 +70,8 @@
     "react-native-vector-icons": "^6.6.0",
     "react-test-renderer": "^16.13.1",
     "rimraf": "^3.0.2",
-    "typescript": "^3.9.5"
+    "typescript": "^3.9.5",
+    "auto-changelog": "*"
```
Comment (pranshuchittora) [inline package.json:74]: Having a specific version would be great.

File: package.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
     "postinstall": "opencollective-postinstall || exit 0",
     "lint": "eslint --ext js,ts .",
     "prettify": "prettier --single-quote --trailing-comma=es5 --write './**/*.md'",
-    "clean-install": "rimraf node_modules && yarn"
+    "clean-install": "rimraf node_modules && yarn",
+    "version": "auto-changelog -p && git add CHANGELOG.md"
```
Comment (pranshuchittora) [inline package.json:26]: IMO adding CHANGELOG to staging is not required. It may interfere with `husky`

Comment (pranshuchittora) [inline package.json:26]:
```suggestion
    "version": "auto-changelog -p"
```

**Review by pranshuchittora — CHANGES_REQUESTED** [review]

**Comment by pranshuchittora** [comment]
Pardon, closing this because it's duplicate -> #2637

**Pull request closed without merging**